docs/release-notes/2.43.0.md

```diff
@@ -15,10 +15,22 @@ This release introduces this and that
 ## Release highlights
 
 
+## New MongoDB collector: CurrentOp
 
+The MongoDB exporter now includes the **CurrentOp** collector, offering visibility into active operations, including the new `mongodb_currentop_query_uptime` metric.
 
+To start the MongoDB exporter with the CurrentOp collector enabled, use the `--enable-all-collectors` flag when adding MongoDB services to your PMM instance.
 
-![!image](../_images/.png)
+```sh
+pmm-admin add mongodb --enable-all-collectors ...
+```
+
+Specifying a limit with `--max-collections` for this collector is not necessary.
+
+!!! warning "Potential high data volume"
+    Enabling all collectors with the `--enable-all-collectors` flag may generate a large volume of data, which could impact disk space usage, especially in high-traffic or large MongoDB environments. Be sure to monitor your disk space after enabling the `--enable-all-collectors` flag.
```

Comment:
Instead of this warning I would mention that to not impact on disk usage we limited collected operations based on their duration. we collect only operations that longer then 1m.